community.md

```diff
@@ -79,6 +79,18 @@ community and for maintaining a queue of development work and as the public
 record of work on the project. We use the mailing lists for development discussions,
 where a lengthy discussions is required.
 
+### Meetings
+
+We host online meetings to provide venues for synchronous discussions about the Arrow project. These discussions usually focus on topics of interest to developers who are contributing to Arrow, but we welcome users of Arrow to join. Currently there are two series of regularly scheduled meetings:
+
+<ul>
+  <li>Arrow community meting, held biweekly</li>
+  <li>Arrow R package development meeting, held biweekly</li>
+</ul>
+
+To receive information about these meetings, subscribe to the <code>dev@</code> mailing list as described above.
```

Comment:
Perhaps something like "If you subscribe to the mailing list then you will get reminders prior to the meeting along with a link to the meeting itself?"

Otherwise, I imagine someone reading this might think they are expected to first subscribe and then also ask where/when the meetings are.